app/views/layouts/angular-bootstrap/_endpoints_angular.html.haml

```diff
@@ -101,6 +101,17 @@
                        "selectpicker-for-select-tag" => "",
                        "prefix"                      => "#{prefix}",
                        "reset-validation-status"     => "#{prefix}_auth_status")
+    .col-md-8{"ng-if" => "emsCommonModel.emstype == 'vmware_cloud'"}
+      - security_protocols = @vmware_cloud_amqp_security_protocols
```

Comment:
If you do choose to merge openstack and vmware amqp security protocols, fix it here as well.